v22.1/use-cloud-storage-for-bulk-operations.md

```diff
@@ -225,6 +225,85 @@ A custom root CA can be appended to the system's default CAs by setting the `clo
 
 </section>
 
+## Storage permissions
+
+This section describes the minimum permissions required to run CockroachDB bulk operations. While we provide the required permissions for Amazon S3 and Google Cloud Storage, the provider's documentation provides detail on the setup process and different options regarding access management.
+
+Depending on the actions a bulk operation performs, it will require different access permissions to a cloud storage bucket. 
+
+This table outlines the actions that each operation performs against the storage bucket:
+
+Operation                   | Permission                   | Description                                                                     
+----------------------------+------------------------------+---------------------------------------------------
+[Backup](backup.html)       | Write<br><br>List                | Write: Backups write the backup data to the bucket/container. During a backup job, a `BACKUP CHECKPOINT` file will be written that tracks the progress of the backup.<br> List: Backups need list access to the files already in the bucket. If the backup is [paused](pause-job.html), or there is a retryable error, the backup will read the `BACKUP CHECKPOINT` file. <br>**Note:** To clean up `BACKUP CHECKPOINT` files that the backup job has written, you would need to also include a delete permission in your bucket policy (e.g., `s3:DeleteObject`). However, delete is **not** necessary for backups  to complete successfully in v22.1 and later.
```

Comment:
extra space: "backups  to"

Reply:
My comments on the 21.2 page apply here as well. In 22.1, List is additionally used during Backup the fine the _latest_ checkpoint. List is additionally used during Restore to find the latest LATEST file.